content/en/real_user_monitoring/platform/connect_rum_and_traces.md

```diff
@@ -540,8 +540,8 @@ Datadog uses the distributed tracing protocol and sets up the HTTP headers below
 `x-datadog-origin: rum`
 : To make sure the generated traces from Real User Monitoring don't affect your APM Index Spans counts.
 
-`x-datadog-sampling-priority: 1`
-: To make sure that the Agent keeps the trace.
+`x-datadog-sampling-priority`
+: Set to `1` by the Real User Monitoring SDK if the trace was sampled, or `0` if it was not.
```

Comment:
The wording feels a little odd here ("...by the Real User Monitoring SDK..."); just trying to keep this consistent with the phrasing used for the other `x-datadog-*` headers.